playground/frontend/playground_components/lib/src/controllers/example_loaders/hive_example_loader.dart

```diff
@@ -37,21 +38,17 @@ class HiveExampleLoader extends ExampleLoader {
   });
 
   @override
-  Sdk? get sdk => _example?.sdk;
+  Sdk? get sdk => descriptor.sdk;
 
-  Future<Example> _getExample() async {
+  @override
+  Future<Example> get future async {
     try {
       final box = await Hive.openBox(descriptor.boxName);
       final Map<String, dynamic> map =
           jsonDecode(box.get(descriptor.snippetId));
       return Example.fromJson(map);
-    } on Exception catch (_) {
+    } on Exception {
       rethrow;
```

Comment:
If this is for breakpoints, add a comment.
Otherwise the whole `try` may be removed.